src/lib.rs

```diff
@@ -1,13 +1,16 @@
+pub mod consensus;
 pub mod sn_membership;
+pub mod vote;
 
-// #[cfg(feature = "bad_crypto")]
-// pub mod bad_crypto;
+#[cfg(feature = "bad_crypto")]
+pub mod bad_crypto;
 // #[cfg(feature = "blsttc")]
 // pub mod blsttc;
-// #[cfg(feature = "ed25519")]
-// pub mod ed25519;
+#[cfg(feature = "ed25519")]
+pub mod ed25519;
```

Comment:
Keep these commented out, they don't work with the current code base

Reply:
It was to appease clippy because they are used for error types, should I comment them back?

Reply:
Oh, sure keep them in then. I wonder why my clippy is not complaining